webapp/middleware/isAdmin.js

```diff
@@ -1,5 +1,5 @@
 export default ({ store, error }) => {
   if (!store.getters['auth/isAdmin']) {
-    return error({ statusCode: 403 })
+    return error({ statusCode: 403, message: 'error-pages.not-authorized' })
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-24T14:15:35Z" title="Friday, January 24th 2020, 3:15:35 pm +01:00">Jan 24, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,5 +1,5 @@
 export default ({ store, error }) => {
   if (!store.getters['auth/isAdmin']) {
-    return error({ statusCode: 403 })
+    return error({ statusCode: 403, message: 'error-pages.not-authorized' })
```


I'm having difficulties with `message` here. It's not the message, it's the translation key.

So, on the consuming side, I would check if there is a translation key for `error.message` (check if it's a key or if the error message is coming from NuxtJS). Just to make sure that we're not trying to translate a default Nuxt error message here.